boards/arc/nsim/nsim_hs6x.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: nsim_hs6x
+name: HS6x nSIM simulator
+type: mcu
+simulation: nsim
+arch: arc
+toolchain:
+  - zephyr
```

Comment:
Could you please add cross-toolchain here so that we may run Twister with cross-tools until ARC64 support is not merged in the Zephyr SDK?

Reply:
Done, thanks!